public/app/plugins/datasource/elasticsearch/types.ts

```diff
@@ -0,0 +1,28 @@
+import { DataQuery, DataSourceJsonData } from '@grafana/ui';
+
+export interface ElasticsearchOptions extends DataSourceJsonData {
+  index: string;
+  timeField: string;
+  esVersion: number;
+  interval: string;
+  timeInterval: string;
+  maxConcurrentShardRequests?: number;
+  logMessageField?: string;
+  logLevelField?: string;
+}
+
+export interface ElasticsearchAggregation {
+  id: string;
+  type: string;
+  settings?: any;
+}
+
+export interface ElasticsearchQuery extends DataQuery {
+  context: string;
```

Comment:
Is this still needed or can we remove it?

Reply:
No longer needed I think. Good catch